src/common/itt/include/openvino/itt.hpp

```diff
@@ -41,6 +41,7 @@ typedef struct handle_ {
  * @cond
  */
 namespace internal {
+bool is_initialized();
```

Comment:
Nit. May be rename `is_initialized` -> `collection_enabled` or `profiling_enabled`?